Cocktails9/Cocktails9/View/BottomRow/ProfileView.swift

```diff
@@ -1,18 +1,206 @@
-//
-//  ProfileView.swift
-//  Cocktails9
-//
-//  Created by Mateja Mitic on 4.12.24..
-//
-
 import SwiftUI
+import PhotosUI
 
 struct ProfileView: View {
+    @State private var user: User = AppDataManager.shared.user ?? User(email: "", username: "Guest", password: "", favoriteCocktails: [])
```

Comment:
I would avoid creating guest user if there is no saved one

Reply:
Guest is only for preview purposes.